doc/cli/npm-version.md

```diff
@@ -68,7 +68,7 @@ Take the following example:
 
     "scripts": {
       "preversion": "npm test",
-      "version": "npm run build && git add -A dist",
+      "version": "npm run build && git add -A",
```

Comment:
The `git add` was correct. If you look at the explanation below this it talks about ONLY adding the things in the `dist` direectory.

Reply:
i think that `git add dist` is enough, without the `-A` switch.